src/cmd/windows/durable_task_monitor.go

```diff
@@ -0,0 +1,150 @@
+// +build windows
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"os/signal"
+	"strings"
+	"sync"
+	"syscall"
+
+	"golang.org/x/sys/windows"
+	"jenkinsci.org/plugins/durabletask/common"
+)
+
+// Launches the script in a new session and waits for its completion.
+func launcher(wg *sync.WaitGroup, exitChan chan bool,
+	executable string, args string, resultPath string, outputPath string,
+	launchLogger *log.Logger, scriptLogger *log.Logger) {
+
+	defer wg.Done()
+	defer common.SignalFinished(exitChan)
+
+	var scriptCmd *exec.Cmd
+	var sysAttr windows.SysProcAttr
+
+	sysAttr.CreationFlags = windows.CREATE_NEW_PROCESS_GROUP
+	if executable == "cmd" {
+		scriptCmd = exec.Command(executable)
+		sysAttr.CmdLine = executable + " " + args
+	} else {
+		fmt.Println(strings.Split(args, ","))
+		scriptCmd = exec.Command(executable, strings.Split(args, ",")...)
```

Comment:
`exec.Command` requires that the first arg be the executable, then the args will follow. Splitting by `,` to handle paths with spaces.

Reply:
but you can have arguments containing `,` ?
Go should be able to understand what commands it was passed and use that array to run a command?

Reply:
At least when I was trying it, it definitely didn't work. `Command`'s signature is: 
`func Command(name string, arg ...string) *Cmd`
https://golang.org/src/os/exec/exec.go?s=6278:6323#L159
`The returned Cmd's Args field is constructed from the command name followed by the elements of arg, so arg should not include the command name itself. For example, Command("echo", "hello"). Args[0] is always name, not the possibly resolved Path.`